pulsar-io/debezium/mysql/src/main/java/org/apache/pulsar/io/debezium/mysql/DebeziumMysqlSource.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.io.debezium.mysql;
+
+import java.util.Map;
+
+import io.debezium.connector.mysql.MySqlConnectorConfig;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.connect.runtime.TaskConfig;
+import org.apache.pulsar.io.core.SourceContext;
+import org.apache.pulsar.io.debezium.PulsarDatabaseHistory;
+import org.apache.pulsar.io.kafka.connect.KafkaConnectSource;
+import org.apache.pulsar.io.kafka.connect.PulsarKafkaWorkerConfig;
+
+/**
+ * A pulsar source that runs
+ */
+@Slf4j
+public class DebeziumMysqlSource extends KafkaConnectSource {
+    static private final String DEFAULT_TASK = "io.debezium.connector.mysql.MySqlConnectorTask";
+    static private final String DEFAULT_CONVERTER = "org.apache.kafka.connect.json.JsonConverter";
+    static private final String DEFAULT_HISTORY = "org.apache.pulsar.io.debezium.PulsarDatabaseHistory";
+    static private final String DEFAULT_TENANT = "public";
```

Comment:
nit: you can use `TopicName.PUBLIC_TENANT` for public tenant and `TopicName.DEFAULT_NAMESPACE` for default namespace.